src/engraving/api/v1/elements.h

```diff
@@ -1542,6 +1538,52 @@ class Chord : public ChordRest
     Q_INVOKABLE void remove(apiv1::EngravingItem* wrapped);
 };
 
+//---------------------------------------------------------
+//   Beam
+///    \since MuseScore 4.7
+//---------------------------------------------------------
+
+class Beam : public EngravingItem
+{
+    Q_OBJECT
+
+    /// The feathering on this beam's left side.
+    API_PROPERTY_T(qreal, growLeft,  GROW_LEFT)
+    /// The feathering on this beam's right side.
+    API_PROPERTY_T(qreal, growRight, GROW_RIGHT)
+    /// Whether this beam is a cross-staff beam.
+    Q_PROPERTY(bool isCrossStaff READ cross)
```

Comment:
I believe it's not only cross-staff, but also in situations like this:

<img width="87" height="52" alt="Image" src="https://github.com/user-attachments/assets/698423e6-d7cc-4fab-8b3c-72aa8c20e283" />

But I haven't thoroughly verified that though

Reply:
Alright, thanks for checking!